src/components/DateRangePickerInputController.jsx

```diff
@@ -41,6 +42,7 @@ const propTypes = forbidExtraProps({
   keepOpenOnDateSelect: PropTypes.bool,
   reopenPickerOnClearDates: PropTypes.bool,
   withFullScreenPortal: PropTypes.bool,
+  minimumNights: PropTypes.number,
```

Comment:
this should probably be a non-negative integer (`nonNegativeInteger` from `airbnb-prop-types`), not just a number